src/client/js/components/main.jsx

```diff
@@ -51,7 +51,7 @@ module.exports = React.createClass({
         {this.isThereAGameToday(nextGame)}
 
         <h2>{ this.props.homeTeam } vs. the fucking { nextGame.opponent }</h2>
-        <h3>{ moment(nextGame.date).format('h:mm a') } { moment(nextGame.date).format('dddd M/D/YYYY') } @ { nextGame.location}</h3>
+        <h3>{ moment(nextGame.time, 'h:ma Z').format('h:mm a') } { moment(nextGame.date).format('dddd M/D/YYYY') } @ { nextGame.location}</h3>
```

Comment:
Sadly, this is the data that they publish. It's kinda a bummer. I've been thinking about how to fix it but it seems... painful. 

http://colorado.rockies.mlb.com/schedule/downloadable.jsp?c_id=col